caching.md

```diff
@@ -0,0 +1,53 @@
+## Introduction
+
+Intake provides easy access data sources from remote / cloud storage, however for large files the 
+cost of downloading files everytime data is read can be extremely high. We have developed a caching 
+strategy to store and manage data sources on the local file system.
```

Comment:
I would add the phrase "download once, read many times" somewhere in here.

Reply:
In a good way?

Reply:
Haha yes!  That's the intake caching slogan.